specification/netapp/resource-manager/Microsoft.NetApp/preview/2017-08-15/netapp.json

```diff
@@ -0,0 +1,1705 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft NetApp",
+    "description": "Microsoft NetApp Azure Resource Provider specification",
+    "version": "2017-08-15",
+    "x-ms-code-generation-settings": {
+      "name": "CloudVolumeManagementClient"
```

Comment:
I don't really understand what you mean. If you mean the `x-ms-code-generation-settings`, then I think I copied that from some document I was pointed out to use as reference. I think it's safe to remove. (I see its not in the cdn.json you're referencing)

Reply:
x-ms-code-generation-settings/name is the final constructor name in the SDK. So it seemed weird to me to have a package called "netapp", with a constructor called "CloudVolumeManagementClient" :)

Do not remove that node, otherwise the constructor will be called "MicrosoftNetAppClient" which is probably not what your want. Renaming "NetAppManagementClient" I guess makes sense.

Reply:
Changing to AzureNetAppFilesManagementClient, since thats the name of service on the site you posted.